client/src/components/__tests__/Reviews.test.js

```diff
@@ -10,9 +11,9 @@ afterEach(() => {
 })
 
 test("should correctly render Reviews given dummy data", () => {
-  const { getByText, getByLabelText } = render(<Reviews reviewData={reviewData} reviews={reviews}/>);
+  const { getByText, getByLabelText } = render(<Reviews reviewData={reviewData} reviews={reviews} product={product}/>);
 
   screen.debug();
```

Comment:
feel free to delete this line once test is working

Reply:
Oh yeah that makes sense. I've been commenting and decommenting it on and off cause the lines it prints are llloooooonnnnggg